<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="Display">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>1</y>
      <width>665</width>
      <height>450</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/play.png</normaloff>:/res/icon/Res/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/pause.png</normaloff>:/res/icon/Res/pause.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/stop.png</normaloff>:/res/icon/Res/stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ToBegin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/tobegin.png</normaloff>:/res/icon/Res/tobegin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ToEnd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/toend.png</normaloff>:/res/icon/Res/toend.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="StepBackOne">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/onebyoneback.png</normaloff>:/res/icon/Res/onebyoneback.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="StepGo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/playonebyone.png</normaloff>:/res/icon/Res/playonebyone.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Slow">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/slow.png</normaloff>:/res/icon/Res/slow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Fast">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/fast.png</normaloff>:/res/icon/Res/fast.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Backword">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/play_backward.png</normaloff>:/res/icon/Res/play_backward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Forword">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/play_forward.png</normaloff>:/res/icon/Res/play_forward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SnapPicture">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/catchpic.png</normaloff>:/res/icon/Res/catchpic.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SetColor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>490</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/color.png</normaloff>:/res/icon/Res/color.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="Process">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>651</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="state">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SetVolume">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>486</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/icon/Res/sound.png</normaloff>:/res/icon/Res/sound.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="Volume">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>484</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="AdjustWave">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>502</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionSnapPicture"/>
    <addaction name="actionLocate"/>
    <addaction name="actionCutFile"/>
    <addaction name="actionMediaInfo"/>
    <addaction name="actionIVS"/>
    <addaction name="actionHwaccel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSnapPicture">
   <property name="text">
    <string>Snap Picture</string>
   </property>
  </action>
  <action name="actionLocate">
   <property name="text">
    <string>Locate</string>
   </property>
  </action>
  <action name="actionCutFile">
   <property name="text">
    <string>Cut File</string>
   </property>
  </action>
  <action name="actionMediaInfo">
   <property name="text">
    <string>Media Info</string>
   </property>
  </action>
  <action name="actionVecticalSync">
   <property name="text">
    <string>VecticalSync</string>
   </property>
  </action>
  <action name="actionIVS">
   <property name="text">
    <string>IVS</string>
   </property>
  </action>
  <action name="actionHwaccel">
   <property name="text">
    <string>Hwaccel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
